conda-forge.yml

```diff
@@ -1,4 +1,5 @@
-bot: {automerge: true}
+# Don't automerge, uvicorn-standard dependencies regularly change and need to edited manually.
+# bot: {automerge: true}
```

Comment:
You can activate grayskull hinting here to have those updates in the bot PR. I'd still keep the automerge off though.

Reply:
grayskull doesn't hint on extra requirements. That's what we would need here.

Reply:
Not sure what you mean by extra. Extra in like, optional but I the python package metadata? Or in external non-python does? The former should work.